viewlib.php

```diff
@@ -261,8 +261,15 @@ function bigbluebuttonbn_view_render_recordings(&$bbbsession, $enabledfeatures,
     );
     // If there are meetings with recordings load the data to the table.
     if ($bbbsession['bigbluebuttonbn']->recordings_html) {
+        $recordingshtml = '';
+        // If opencast recoding config is set and checks if opencast is available for the current course.
+        if ((boolean) \mod_bigbluebuttonbn\locallib\config::get('oc_recording')
+            && bigbluebuttonbn_check_opencast($bbbsession['course']->id)) {
+            $recordingshtml .= bigbluebuttonbn_output_recording_opencast($bbbsession['course']->id);
+        }
+        $recordingshtml .= bigbluebuttonbn_output_recording_table($bbbsession, $recordings)."\n";
         // Render a plain html table.
```

Comment:
After injecting the Opencast functionality here, the existing comments should be adapted.
```suggestion
            // Render Opencast recordings.
            $recordingshtml .= bigbluebuttonbn_output_recording_opencast($bbbsession['course']->id);
        }
        // Render BigBlueButton recordings table.
        $recordingshtml .= bigbluebuttonbn_output_recording_table($bbbsession, $recordings)."\n";
        // Return recordings HTML code.
```